lxd/iptables/xtables.go

```diff
@@ -0,0 +1,337 @@
+package iptables
+
+import (
+	"encoding/hex"
+	"fmt"
+	"net"
+	"strings"
+
+	deviceConfig "github.com/lxc/lxd/lxd/device/config"
+	"github.com/lxc/lxd/lxd/util"
+	"github.com/lxc/lxd/shared"
+)
+
+// XTables is an implmentation of LXD firewall using {ip, ip6, eb}tables
+type XTables struct{}
+
+// Lower-level Functions
+
+// NetworkClear removes network rules.
+func (xt XTables) NetworkClear(protocol string, comment string, table string) error {
+	return NetworkClear(protocol, comment, table)
+}
+
+// InstanceClear removes rules all rules for the given instance.
+func (xt XTables) InstanceClear(protocol string, comment string, table string) error {
```

Comment:
Same comment wrt protocol renaming to family.
Also, I think the comment should go as last argument.